components/automate-cli/pkg/verifyserver/server/api/v1/systemresources.go

```diff
@@ -0,0 +1,83 @@
+package v1
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/chef/automate/components/automate-cli/pkg/verifyserver/constants"
+	"github.com/chef/automate/components/automate-cli/pkg/verifyserver/response"
+	"github.com/gofiber/fiber/v2"
+)
+
+func GetNodeTypeMap() map[string]constants.NodeType {
```

Comment:
Please make all these validations functions as private

Reply:
Instead of calling a function everywhere you can create a global variable and use it ?

Reply:
Done 1st one.For second i think we should allocate memory for map only when its needed rather than creating global map.